src/languages/pyret/ast.tsx

```diff
@@ -1506,7 +1516,7 @@ export class Table extends AST.ASTNode {
 		this.rows.forEach((aRow, index) => {
 			let cellElements = [];       
 			rowBranches.push(<span className="">
-        <DropTarget />
+        <DropTarget field="rows"/>
```

Comment:
@schanzer another bug fix. I haven't looked into how all the `DropTarget` stuff works so I'm hoping I used the right value for the `field` prop? This is another case where improved api documentation would be helpful

Reply:
Yes, this looks right. A `DropTarget` can infer the ASTNode in which it exists, but it needs to know which constructor field of that node it corresponds to. This is used in the core CMB library to generate synthetic edits (see /src/edits/`) which are then converted into real CodeMirror change objects.

This allows a block developer to ignore the order of the constructor fields, putting any_thing_ any_where_.